Sources/MessagingInApp/MessagingInApp.swift

```diff
@@ -4,13 +4,22 @@ import Foundation
 import Gist
 
 public protocol MessagingInAppInstance: AutoMockable {
-    func initialize(organizationId: String)
+    // sourcery:Name=initialize
+    func initialize()
     // sourcery:Name=initializeEventListener
+    func initialize(eventListener: InAppEventListener)
+
+    @available(*, deprecated, message: "Parameter organizationId no longer being used. Remove the parameter from your function call to migrate to new function.")
+    // sourcery:Name=initializeOrganizationId
+    func initialize(organizationId: String)
+
+    @available(*, deprecated, message: "Parameter organizationId no longer being used. Remove the parameter from your function call to migrate to new function.")
+    // sourcery:Name=initializeOrganizationIdEventListener
```

Comment:
Since `event listener` hasn't been released to clients, I don't think we should expose it to the constructor we are deprecating. 

To discourage clients from using this field, we should just provide event listeners to the new method.